### PR TITLE
[WIP] Add health check support to Route53

### PR DESCRIPTION
**Pull request by cachedout**
Needs a little additional testing first.

### CODE REVIEW

File: salt/modules/boto_route53.py [hunk 1]
```diff
@@ -319,6 +319,37 @@ def create_zone(zone, private=False, vpc_id=None, vpc_region=None, region=None,
...
+                                               string_match=None,
+                                               request_interval=30,
+                                               failure_threshold=3)
+
+    if region is None:
+        region = 'universal'
+
+    conn = _get_conn(region=region, key=key, keyid=keyid, profile=profile)
+
+    conn.create_health_check(hc_)
```
Comment (ryan-lane) [inline salt/modules/boto_route53.py]: Error handling?

Comment (cachedout) [inline salt/modules/boto_route53.py]: I'm just POC'ing this ATM, but yes, absolutely. I'll get some in. Thanks, @ryan-lane

**Comment by rickh563** [comment]
ZD-1261

**Comment by cachedout** [comment]
Re-opened. Still WIP.

**Comment by tkwilliams** [comment]
@cachedout  The boto3_route53 module handles passing in healthchecks nicely, but the method used is not directly portable to the boto2 module - boto3 follows the lead of the AWS API itself, and simply accepts a `ChangeBatch` json structure, which in turn defines a list of `Changes` records, which are to be applied (or failed) atomically.  I believe the boto2 module took a much more parameterized approach, so there would be no (less than hideous) way to port it back to boto2_route53 :(

**Comment by rallytime** [comment]
This was replaced with #44950. Closing.

**Pull request closed without merging**